packages/react-component-library/src/components/TabSet/ScrollButton.tsx

```diff
@@ -30,6 +30,7 @@ export const ScrollButton: React.FC<ScrollButtonProps> = ({
     <button
       className={classes}
       onClick={onClick}
+      aria-hidden
```

Comment:
I am wondering if we actually want this hidden or to give it `aria-label`? Are you thinking that the user still has full functionality with a screenreader, without using the scroll buttons?

Reply:
Yeah my assumption was that they wouldn't interact with this element anyway because of the screen reader.